tests/model_registry/model_catalog/utils.py

```
@@ -351,3 +351,33 @@ def validate_model_catalog_configmap_data(configmap: ConfigMap, num_catalogs: in
     assert len(catalogs) == num_catalogs, f"{configmap.name} should have {num_catalogs} catalog"
     if num_catalogs:
         validate_default_catalog(catalogs=catalogs)
+
+
+def get_models_from_api(
+    model_catalog_rest_url: list[str],
+    model_registry_rest_headers: str,
+    page_size: int = 100,
+    source_label: str = None,
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)